src/codemagic/tools/firebase/firebase.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+from __future__ import annotations
+
+import argparse
+import json
+
+from codemagic import cli
+from codemagic.firebase.api_client import FirebaseApiClient
+
+from .action_groups import ReleasesActionGroup
+from .actions import GetLatestBuildVersionAction
+from .arguments import FirebaseArgument
+
+
+@cli.common_arguments(FirebaseArgument.PROJECT_ID, FirebaseArgument.FIREBASE_SERVICE_ACCOUNT_CREDENTIALS)
+class Firebase(cli.CliApp, GetLatestBuildVersionAction, ReleasesActionGroup):
```

Comment:
It is expected that more actions and action groups are added to this. Hence it is better to format parents on separate lines from the getgo.